library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionListFragment.kt

```diff
@@ -63,6 +75,12 @@ internal class TransactionListFragment :
                     if (transactionTuples.isEmpty()) View.VISIBLE else View.GONE
             }
         )
+        RepositoryProvider.transaction().getAllTransactions().observe(
+            viewLifecycleOwner,
+            Observer {
```

Comment:
For this particular case we can just return a list of items once without `LiveData` and observers.
Like when user clicks on `Export` we just call `viewModel.getAllTransactions()` and that is it. There is no need in maintaining LiveData object, since user fetches items only once for his export action.

Reply:
agreed hence updated.